ci-operator/step-registry/cluster-kube-scheduler-operator/e2e-ote/cluster-kube-scheduler-operator-e2e-ote-commands.sh

```
@@ -0,0 +1,186 @@
+
+# Create output directories for each test suite
+mkdir -p "${ARTIFACT_DIR}/junit-operator-serial"
+mkdir -p "${ARTIFACT_DIR}/junit-operator-parallel"
+mkdir -p "${ARTIFACT_DIR}/junit-preferred-host-serial"
+
+# Run operator/serial test suite
+echo "========================================"
+echo "$(date) - Running operator/serial test suite..."
+echo "========================================"
+TEST_SUITE="openshift/cluster-kube-scheduler-operator/operator/serial"
+openshift-tests run "${TEST_SUITE}" ${TEST_ARGS} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)